calculator/src/App.js

```diff
@@ -1,25 +1,65 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState, useEffect } from 'react';
+
+import Container from 'react-bootstrap/Container'
+import Col from 'react-bootstrap/Col'
+import Row from 'react-bootstrap/Row'
+
+import "./app.css"
 
 function App() {
+
+  const [partyNumber, setPartyNumber] = useState(5)
+  const [fullCheckAmount, setFullCheckAmount] = useState(0)
+  const [tip, setTip] = useState(0)
+
+  const addOnePerson = () => setPartyNumber(partyNumber + 1)
```

Comment:
Simple, clean function... but I'd suggest shorter names (ie addPerson or addOne) since this functionality will not change based on how you've written the code, it will always be one added unless you have the numerical value adjusted by the state or something elsewhere.

Reply:
agree